node/src/reactor.rs

```diff
@@ -223,6 +227,9 @@ where
 
     /// An accurate, possible TSC-supporting clock.
     clock: Clock,
+
+    /// Last queue dump timestamp
+    last_queue_dump: Option<Timestamp>,
```

Comment:
Since you don't make a use of the value of the last queue dump, this _could_ be just a boolean, but I understand that having the actual timestamp may be more useful in the future.